redislock_test.go

```diff
@@ -236,17 +247,28 @@ func TestLock_Release_expired(t *testing.T) {
 }
 
 func TestLock_Release_not_own(t *testing.T) {
+	lockKey := getLockKey()
 	ctx := context.Background()
 	rc := redis.NewClient(redisOpts)
-	defer teardown(t, rc)
+	defer teardown(t, rc, lockKey)
 
-	lock := quickObtain(t, rc, time.Hour)
+	lock := quickObtain(t, rc, lockKey, time.Hour)
 	defer lock.Release(ctx)
 
+	realKey := "redislock_" + lockKey
```

Comment:
Again - the lib should better not assume any prefixes on its own - just use what user provides.

Doesn't prevent us from merging and then polishing it ourselves.

Reply:
Removed